src/Sylius/Behat/Context/Cli/CancelUnpaidOrdersContext.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Cli;
+
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Webmozart\Assert\Assert;
+use Behat\Behat\Context\Context;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+final class CancelUnpaidOrdersContext implements Context
+{
+    private const CANCEL_UNPAID_ORDERS_COMMAND = 'sylius:cancel-unpaid-orders';
+
+    private Application $application;
+
+    private ?CommandTester $commandTester = null;
+
+    private OrderRepositoryInterface $orderRepository;
+
+    public function __construct(KernelInterface $kernel, OrderRepositoryInterface $orderRepository)
+    {
+        $this->application = new Application($kernel);
+        $this->orderRepository = $orderRepository;
+    }
+
+    /**
+     * @Then only unpaid order with number :orderNumber should be canceled
+     */
+    public function runCancelUnpaidOrdersCommand(string $orderNumber): void
```

Comment:
```suggestion
    public function onlyUnpaidOrderWithNumberShouldBeCancelled(string $orderNumber): void
```
and try to use step definition as the name of the method in other places